sparse/tpls/KokkosSparse_spmv_tpl_spec_decl.hpp

```diff
@@ -78,19 +74,11 @@ void spmv_cusparse(const Kokkos::Cuda& exec, Handle* handle, const char mode[],
   // matrices. After 11.2.1, the default is performant for imbalanced matrices,
   // and ALG2 now means something else. CUDA >= 11.2.1 corresponds to
   // CUSPARSE_VERSION >= 11402.
```

Comment:
```suggestion
After CUDA 11.2.1, the default is performant for imbalanced matrices
```

Old comment can be deleted as well, though the info might be a helpful reference reminder? not blocking